roles/common/tasks/ssl.yml

```diff
@@ -7,6 +7,14 @@
             dest=/usr/local/share/ca-certificates/{{ endpoints.main }}.crt
   notify: refresh CAs
 
+- name: Install any additional CA certificates
+  copy:
+    content: "{{ item.content | default(omit) }}"
+    dest: "/usr/local/share/ca-certificates/{{ item.name }}.crt"
+    src: "{{ item.src | default(omit) }}"
+  with_items: "{{ cacerts | default([]) }}"
```

Comment:
We use the hash merge setting for Ansible, so I believe that if you defined ssl.cacerts in your host vars it would get merged into the default ssl hash, and you get the union of the two. Please test?